src/app/calendar/page.tsx

```diff
@@ -1,12 +1,18 @@
 import { Metadata } from "next";
+import SampleCalendar from "./clientPage";
 import React from "react";
 
-const Calendar: React.FC<{}> = () => {
-    return <h1> Calendar Page </h1>;
-};
-
 export const metadata: Metadata = {
     title: "Calendar",
 };
 
-export default Calendar;
+const CalendarPage: React.FC<{}> = () => {
+    return (
+        <>
+            <h1>Calendar Page</h1>
+            <SampleCalendar />;
```

Comment:
There's an extra semi-colon on this line that's getting rendered at the bottom of the calendar